doc/qbk/quickref.xml

```diff
@@ -55,6 +55,11 @@
           <member><link linkend="json.ref.boost__json__has_value_from">has_value_from</link></member>
           <member><link linkend="json.ref.boost__json__has_value_to">has_value_to</link></member>
           <member><link linkend="json.ref.boost__json__is_deallocate_trivial">is_deallocate_trivial</link></member>
+          <member><link linkend="json.ref.boost__json__is_string_like">is_string_like</link></member>
+          <member><link linkend="json.ref.boost__json__is_sequence_like">is_sequence_like</link></member>
+          <member><link linkend="json.ref.boost__json__is_map_like">is_map_like</link></member>
+          <member><link linkend="json.ref.boost__json__is_null_like">is_null_like</link></member>
+          <member><link linkend="json.ref.boost__json__is_tuple_like">is_tuple_like</link></member>
```

Comment:
These are not in alphabetical order, was this intended?

Reply:
No, it was a mistake